build/charts/antrea/values.yaml

```diff
@@ -298,3 +298,7 @@ testing:
   coverage: false
   simulator:
     enable: false
+
+nodeport:
+  enable: false
+  port: 32767
```

Comment:
* there is no documentation for this
* the name `nodeport` is to vague
* can we use absence of value for `port` / a zero value as meaning "don't enable NodePort" instead of having an extra boolean flag?
* this should be under `controller:` IMO

Reply:
The documentation on using NodePort for antrea-controller will be added in #3963

Reply:
Updated the PR to remove to consider zero value as disabling NodePort.
For the documentation, we have leveraged the existing framework to ssh into the VM (i.e. external node). So I don't think we need a separate/documentation for the VM test.

Reply:
> For the documentation, we have leveraged the existing framework to ssh into the VM (i.e. external node). So I don't think we need a separate/documentation for the VM test.

I don't understand this comment. I was pointing to the lack of comment to document this Helm value.

Reply:
Updated it.